joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/astcreation/AstForTypesCreator.scala

```diff
@@ -230,7 +230,7 @@ trait AstForTypesCreator(implicit withSchemaValidation: ValidationMode) { this:
     registerType(typeFullName)
 
     val astParentType     = methodAstParentStack.head.label
-    val astParentFullName = methodAstParentStack.head.properties("FULL_NAME").toString
+    val astParentFullName = methodAstParentStack.head.properties(PropertyNames.FULL_NAME).toString
```

Comment:
This is unfortunately quite bad: It assembles all properties into a dictionary before extracting the one it needs.

This is not a new problem, but it does suck :(

Reply:
we could define `NewNode.property(name)` in the codegen. after the flatgraph migration. sounds good?